nsqd/http.go

```diff
@@ -226,6 +231,32 @@ func (s *httpServer) getTopicFromQuery(req *http.Request) (url.Values, *Topic, e
 	return reqParams, s.ctx.nsqd.GetTopic(topicName), nil
 }
 
+func (s *httpServer) doLookup(req *http.Request) (interface{}, error) {
```

Comment:
:+1: to that error - related I think /health or /stats should report gossip status... thoughts?

I'll look into code re-use...